helm/dex-app/templates/dex-secret.yaml

```diff
@@ -8,6 +8,7 @@ stringData:
   config.yaml: |-
     web:
       http: 0.0.0.0:5556
+      allowedOrigins: ['*']
```

Comment:
This is to allow cors requests

Reply:
This might give us some problems in security audits.

Reply:
I added specific URLs, let's see if it still works

Reply:
It works 👍